weather-app/src/app/app.component.html

```diff
@@ -2,3 +2,5 @@
   <router-outlet></router-outlet>
 </main>
 <app-footer></app-footer>
+<app-favorite></app-favorite>
```

Comment:
i think app-favorite must position inside a home component

Reply:
and one more things, create favorite-list and favorite-item components